include/deal.II/base/config.h.in

```diff
@@ -150,6 +150,9 @@
 #cmakedefine DEAL_II_PETSC_WITH_HYPRE
 #cmakedefine DEAL_II_PETSC_WITH_MUMPS
 
+/* cmake/modules/FindSUNDIALS.cmake */
+#cmakedefine DEAL_II_SUNDIALS_WITH_IDAS
+
```

Comment:
@juan42 That gets transformed into `DEAL_II_SUNDIALS_WITH_IDAS` and is exported in `config.h.in`. That way the preprocessor definition is available during compilation.